### DISCUSSION TITLE
mfsu模式下的esbuild 只支持用户配置extraPostCSSPlugins吗？

### DISCUSSION THREAD

**Discussion by notcold** [umijs/umi#9369] [Q&A]
https://github.com/umijs/umi/blob/c922d954048afcc133bf7a533a9a6ac5c0e19539/packages/bundler-webpack/src/dev.ts#L69

这里depBuildConfig默认只允许项下传extraPostCSSPlugins？导致 esbuild模式获取不到额外配置，无法配置专门的方式解决对依赖的处理

**Comment by fz6m** [comment]
给一个复现看看
